ContactDetailsApi/V2/Factories/EntityFactory.cs

```diff
@@ -29,7 +32,7 @@ public static ContactDetails ToDomain(this ContactDetailsEntity databaseEntity)
                 TargetType = databaseEntity.TargetType,
                 ContactInformation = contactInformation,
                 SourceServiceArea = databaseEntity.SourceServiceArea,
-                CreatedBy = databaseEntity.CreatedBy,
+                CreatedBy = databaseEntity?.CreatedBy,
```

Comment:
Since a null check is done for `databaseEntity` in line 17, this conditional null check operator `.?` is redundant for the `CreatedBy` property